test/commands/force/org/create.test.ts

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  Aliases,
+  Config,
+  Lifecycle,
+  Org,
+  SandboxEvents,
+  SandboxProcessObject,
+  SandboxUserAuthResponse,
+  SfdxProject,
+} from '@salesforce/core';
+import { fromStub, stubInterface, stubMethod } from '@salesforce/ts-sinon';
+import { IConfig } from '@oclif/config';
+import * as sinon from 'sinon';
+import { expect } from '@salesforce/command/lib/test';
+import { UX } from '@salesforce/command';
+import { Duration } from '@salesforce/kit';
+import { Create } from '../../../../src/commands/force/org/beta/create';
+
+describe('org:create', () => {
+  const sandbox = sinon.createSandbox();
+  const oclifConfigStub = fromStub(stubInterface<IConfig>(sandbox));
+
+  // stubs
+  let resolveProjectConfigStub: sinon.SinonStub;
+  let createSandboxStub: sinon.SinonStub;
+  let uxWarnStub: sinon.SinonStub;
+  let uxLogStub: sinon.SinonStub;
+  let uxTableStub: sinon.SinonStub;
+  let uxStyledHeaderStub: sinon.SinonStub;
+  let cmd: TestCreate;
+
+  class TestCreate extends Create {
+    public async runIt() {
+      await this.init();
+      return this.run();
+    }
+    public setOrg(org: Org) {
+      this.org = org;
+    }
+    public setProject(project: SfdxProject) {
+      this.project = project;
+    }
+  }
+
+  const createCommand = async (params: string[]) => {
+    cmd = new TestCreate(params, oclifConfigStub);
+    stubMethod(sandbox, cmd, 'assignProject').callsFake(() => {
+      const sfdxProjectStub = fromStub(
+        stubInterface<SfdxProject>(sandbox, {
+          resolveProjectConfig: resolveProjectConfigStub,
+        })
+      );
+      cmd.setProject(sfdxProjectStub);
+    });
+
+    createSandboxStub = stubMethod(sandbox, cmd, 'createSandbox').resolves();
+
+    uxWarnStub = stubMethod(sandbox, UX.prototype, 'warn');
+    uxLogStub = stubMethod(sandbox, UX.prototype, 'log');
+    uxStyledHeaderStub = stubMethod(sandbox, UX.prototype, 'styledHeader');
+    uxTableStub = stubMethod(sandbox, UX.prototype, 'table');
+    stubMethod(sandbox, cmd, 'assignOrg').callsFake(() => {
+      const orgStubOptions = {};
+
+      const orgStub = fromStub(stubInterface<Org>(sandbox, orgStubOptions));
+      cmd.setOrg(orgStub);
+    });
+    return cmd;
+  };
+
+  describe('scratch org', () => {});
```

Comment:
I think this file should be sandboxCreate.test.ts and scratch org doesn't have to be in the same file.